rtengine/camconst.json

```diff
@@ -2537,6 +2537,14 @@ Camera constants:
         }
     },
 
+    { // Quality C
+        "make_model": "Panasonic DC-GH6",
+        "dcraw_matrix": [ 7949, -3491, -710, -3435, 11681, 1977, -503, 1622, 5065 ], // Adobe DNG Converter
+        "ranges": {
+            "white": 65535
+        }
```

Comment:
The decoder already has the white level for this camera.
```white levels: R:65535 G1:65535 B:65535 G2:65535 (provided by dcraw)```

Reply:
Right.

If useful this isn't directly provided by any metadata but calculated from tiff_bps (16 in this case): https://github.com/Beep6581/RawTherapee/blob/5a25619bd50d42059abe9818420f639b6d10e3bf/rtengine/dcraw.cc#L10720

Reply:
Similar to the edit to my comment in the other Panasonic camconst PR, Libraw has the color matrix and white level. We should consider copying the new Libraw constants for all cameras into dcraw.cc.
https://github.com/LibRaw/LibRaw/blob/158e635e5e80a95dfa402b0a79a7167922c1d4ec/src/tables/colordata.cpp#L1387-L1388